pages/doc.js

```diff
@@ -94,48 +77,90 @@ export default class Documentation extends Component {
     })
   }
 
-  getLinkHref = (section, file) => {
-    const sectionSlug =
-      sidebar[section].slug || kebabCase(sidebar[section].name)
-    const fileSlug = file ? kebabCase(file.slice(0, -3)) : undefined
-    return `/doc/${compact([sectionSlug, fileSlug]).join('/')}`
+  getLinkHref = (section, subsection = null, file = null) => {
+    let sect = sidebar[section]
+    let removeExtFunc = filename =>
```

Comment:
let's just rename this function in the SideHelper in the first place

Reply:
fix it

Reply:
so, is it fixed? forgot to push the changes?

Reply:
Sorry, were not careful, renamed to SidebarHelper, now fix